soci/soci_index.go

```diff
@@ -255,7 +253,7 @@ func buildSociLayer(ctx context.Context, cs content.Store, desc ocispec.Descript
 		Digest:         ztocDesc.Digest.String(),
 		OriginalDigest: desc.Digest.String(),
 		Type:           ArtifactEntryTypeLayer,
-		Location:       path.Join(config.SociIndexDirectory, desc.Digest.String()),
+		Location:       desc.Digest.String(),
```

Comment:
Does location make sense anymore? The only place we're using this metadata is in the CLI which assumes the content is in the containerd content store.

Reply:
Yeah, I was hesitant to remove it entirely in case it broke something. I will create another PR to remove it.